lumen_runtime/src/otp/erlang/time_offset_1.rs

```diff
@@ -0,0 +1,26 @@
+// wasm32 proptest cannot be compiled at the same time as non-wasm32 proptest, so disable tests that
+// use proptest completely for wasm32
+//
+// See https://github.com/rust-lang/cargo/issues/4866
+#[cfg(all(not(target_arch = "wasm32"), test))]
```

Comment:
You're not using `proptest` for the `time_offset_1` tests.  You need to add `badarg` tests for improper units, which would could use proptest or remove the comment and weaken the config to `#[cfg(test)]` if you're not going to use proptest for the badarg tests.

Reply:
Thanks, I weakened the config to `#[cfg(test)]`